src/conversation.ts

```diff
@@ -361,8 +361,8 @@ export class Conversation {
      * conversation.publish("my-event", {payload: "bob"});
```

Comment:
Is the example out of date?

Reply:
Nope, still fine :slightly_smiling_face: